packages/Popover/src/components/Content/Content.js

```diff
@@ -100,7 +100,7 @@ const Content = React.forwardRef((props, ref) => {
       onMouseOut={handleMouseEvent}
       onMouseOver={handleMouseEvent}
       style={contentStyles}
-      tabIndex={isOpen ? 0 : -1}
+      tabIndex="-1"
```

Comment:
Not important to toggle this. It never needs to be in the natural tab order and can always be focussed programmatically when it's `-1`.